scylla-cdc-replicator/src/main.rs

```diff
@@ -1,29 +1,58 @@
 mod replication_tests;
 pub mod replicator_consumer;
 
-use replicator_consumer::ReplicatorConsumerFactory;
-use scylla::SessionBuilder;
-use std::error::Error;
-use std::sync::Arc;
 use std::time::Duration;
 
+use clap::Parser;
+
+use replicator_consumer::Replicator;
+
+#[derive(Parser)]
+struct Args {
+    #[clap(short, long)]
+    keyspace: String,
+
+    #[clap(short, long)]
+    table: String,
+
+    #[clap(short, long)]
+    source: String,
+
+    #[clap(short, long)]
+    destination: String,
```

Comment:
A short description of the flags that shows on `--help` in the CLI would be appreciated.